ads/_config.js

```diff
@@ -216,6 +216,14 @@ export const adConfig = {
     renderStartImplemented: true,
   },
 
+  aja: {
+    renderStartImplemented: true,
+    prefetch: 'https://cdn.as.amanad.adtdp.com/sdk/asot-v2.js',
+    preconnect: [
```

Comment:
You do not need to preconnect if url is in prefetch

Reply:
thx. fixed it!